index.html

```diff
@@ -0,0 +1,224 @@
+        <p>Confira alguns dos meus trabalhos realizados!</p>
+
+        <div id="linha">
+            <hr />
+        </div>
+
+        <nav id="trabalhos">
+            <ul>
+                <li style="display:inline"><a href="#">Tudo</a></li>
+                <li style="display:inline"><a href="#">Website</a></li>
+                <li style="display:inline"><a href="#">Template</a></li>
+                <li style="display:inline"><a href="#">Tema</a></li>
+            </ul>
+
+            <div class="jobs">
+                <div class="jobs-col1">
+                    <img src="./images/Portfolio01.png" alt="portifolio1">
+                    <p>Trabalho1</p>
+                </div>
+
+                <div class="jobs-col2">
+                    <img src="./images/Portfolio02.png" alt="Portfolio02">
+                    <p>Trabalho2</p>
+                </div>
+
+                <div class="jobs-col1">
+                    <img src="./images/Portfolio03.png" alt="Portfolio03">
+                    <p>Trabalho3</p>
+                </div>
+    </section>
+
+    <section id="quem-sou-eu">
+        <h1>Quem sou eu?</h1>
+        <p>Grande incentivadora de movimentos de apoio ao software livre.</p>
+        <div id="linha">
+            <hr />
+        </div>
+        <figure>
+            <img src="./images/photo.png">
+            <figcaption>Daiane Alves</figcaption>
+        </figure>
+
+        <table>
+            <tbody>
+                <tr>
+                    <td>
+                        <a href="https://www.facebook.com/DaianeAlvesRJ"><img src="./images/facebook.png" title="facebook" />
+                        </a>
+                    </td>
+
+                    <td>
+                        <a href="https://www.instagram.com/daianealvesrj/"><img src="./images/instagram.png" title="instagram" />
+                        </a>
+                    </td>
+
+                    <td>
+                        <a href="https://twitter.com/daianealvesrj"><img src="./images/twitter.png" title="twitter" />
+                        </a>
+                    </td>
+
+                    <td>
+                        <a href="https://www.linkedin.com/in/daianealvesrj/"><img src="./images/linkedin.png" title="linkedin" />
+                        </a>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+
+        <div id="sobre-mim">
+            <h1>Habilidades</h1>
+            COLOCAR UM TEXTO AQUI!
+        </div>
+
+        <div id="habilidades">
+            <ul>
+                <li>Javascript</li>
+                <li>HTML e CSS</li>
+                <li>Jquery</li>
+                <li>Wordpress</li>
+            </ul>
+        </div>
+    </section>
+
+    <section id="contato">
+        <h1>Contato</h1>
+        <p>Precisando de ajuda para tirar aquela ideia do papel? Vamos bater um papo?</p>
+        <div id="linha">
+            <hr />
+        </div>
+        <p>daiane@lt.coop.br</p>
```

Comment:
Você pode facilitar a vida do teu usuário colocando seu email clicável.
```suggestion
        <a href="mailto:daiane@lt.coop.br">daiane@lt.coop.br</a>
```